dvc/stage/exceptions.py

```diff
@@ -124,5 +124,5 @@ class InvalidStageName(DvcException):
     def __init__(self,):
         super().__init__(
             "Stage name cannot contain invalid characters: "
-            "'\\', '/', '@' and ':'."
+            "'\\', '/', and ':'."
```

Comment:
Created https://github.com/iterative/dvc/issues/3700 to discuss this separately. This will allow `@` for now in the stage name.

Reply:
Let's not allow @ for now, it is a useful character that will def come in handy at some point.